public/js/map.js

```diff
@@ -1,27 +1,28 @@
+import 'whatwg-fetch';
+
+const L = require('mapbox.js');
+const mapStyles = require('./mapStyles');
 const config = require('../../frontendconfig.json');
 const error = require('./error');
 
-import 'whatwg-fetch';
-
 let map;
 let startMarker;
 let endMarker;
 let path;
 let initialCenter;
 
-const L = require('mapbox.js');
-
 // Setup mapbox
 L.mapbox.accessToken = config.mapboxAccessToken;
+const MAPBOX_DATASETS_API = 'https://api.mapbox.com/datasets/v1/tahoebike';
 
 // Setup layers
 const class1Layer = L.mapbox.featureLayer();
 const class2Layer = L.mapbox.featureLayer();
 const class3Layer = L.mapbox.featureLayer();
 const winterLayer = L.mapbox.featureLayer();
-const bikeParkingLayer = L.layerGroup();
-const bikeShopsLayer = L.layerGroup();
-const constructionLayer = L.layerGroup();
+const bikeParkingLayer = L.mapbox.featureLayer();
+const bikeShopsLayer = L.mapbox.featureLayer();
+const constructionLayer = L.mapbox.featureLayer();
```

Comment:
Got any insight into the key differences between `layerGroup` vs `featureLayer`?

Reply:
Yeah a layerGroup is an entity to collect several entities in a group. Previously we would add each `Marker` to the map and then group them with the layerGroup. Now we just add a featurelayer and one-shot dump geojson to it.